### PR TITLE
Ionutpoplacean branch

### PR DESCRIPTION
**Pull request by ionut0**
added Ionut Poplacean tasks

### CODE REVIEW

File: ionut-poplacean-dir/ionutpoplacean.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Ionut Poplacean created this file during Lab 5
+second commit
```
Comment (CenteaPaul) [inline ionut-poplacean-dir/ionutpoplacean.txt:2]: reviewed by paulCentea

**Review by CenteaPaul — CHANGES_REQUESTED** [review]

File: ionut-poplacean-dir/ionutpoplacean.txt (hunk 1 above)
Comment (AndreiPatrausanu) [inline ionut-poplacean-dir/ionutpoplacean.txt:2]: Please add "reviewed by Andrei Patrausanu"

**Review by AndreiPatrausanu — CHANGES_REQUESTED** [review]

File: ionut-poplacean-dir/ionutpoplacean.txt (hunk 1 above)
Comment (ionut0) [reply ionut-poplacean-dir/ionutpoplacean.txt:2]: done

Comment (ionut0) [reply ionut-poplacean-dir/ionutpoplacean.txt:2]: done

**Review by CenteaPaul — APPROVED** [review]

**Review by AndreiPatrausanu — APPROVED** [review]

**Pull request merged**